### PR TITLE
some minor code cleanup

### PR DESCRIPTION
**Pull request by unid15**
tested & works

### CODE REVIEW

File: code/ATMOSPHERICS/components/binary_devices/valve.dm [hunk 1]
```diff
@@ -80,9 +80,6 @@
 /obj/machinery/atmospherics/binary/valve/attack_ai(mob/user as mob)
 	return
 
-/obj/machinery/atmospherics/binary/valve/attack_paw(mob/user as mob)
```
Comment (unid15) [inline code/ATMOSPHERICS/components/binary_devices/valve.dm:84]: /obj/machinery/attack_paw() already does this

**Comment by Exxion** [comment]
The lists you replaced with `organs_by_name` were not identical, so that'll have some gameplay impact
Very minor and it probably changes it to what was originally intended, though

File: code/modules/genetics/side_effects.dm [hunk 2]
```diff
@@ -5,44 +5,46 @@
...
+/datum/genetics/side_effect/genetic_burn/start(mob/living/carbon/human/H)
+	H.emote("me", 1, "starts turning very red..")
 
-	finish(mob/living/carbon/human/H)
-		if(!H.reagents.has_reagent(DEXALIN))
-			for(var/organ_name in list(LIMB_CHEST,LIMB_LEFT_ARM,LIMB_RIGHT_ARM,LIMB_RIGHT_LEG,LIMB_LEFT_LEG,LIMB_HEAD,LIMB_GROIN))
-				var/datum/organ/external/E = H.get_organ(organ_name)
+/datum/genetics/side_effect/genetic_burn/finish(mob/living/carbon/human/H)
+	if(!H.reagents.has_reagent(DEXALIN))
+		for(var/organ_name in H.organs_by_name)
```
Comment (unid15) [inline code/modules/genetics/side_effects.dm:28]: I started replacing some of the lists like _list(LIMB_CHEST,LIMB_LEFT_ARM,LIMB_RIGHT_ARM,LIMB_RIGHT_LEG,LIMB_LEFT_LEG,LIMB_HEAD,LIMB_GROIN)_ with the mob's actual list of organs.
Sometimes this does affect gameplay - as some of those lists don't include every organ in the mob. In most cases it's not intentional

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 3]
```diff
@@ -61,7 +61,6 @@
 	var/parrot_state = PARROT_WANDER //Hunt for a perch when created
 	var/parrot_sleep_max = 25 //The time the parrot sits while perched before looking around. Mosly a way to avoid the parrot's AI in life() being run every single tick.
 	var/parrot_sleep_dur = 25 //Same as above, this is the var that physically counts down
-	var/parrot_dam_zone = list(LIMB_CHEST, LIMB_HEAD, LIMB_LEFT_ARM, LIMB_LEFT_LEG, LIMB_RIGHT_ARM, LIMB_RIGHT_LEG) //For humans, select a bodypart to attack
```
Comment (unid15) [inline code/modules/mob/living/simple_animal/parrot.dm:64]: not used

**Comment by unid15** [comment]
yep, but it's only because the oldcoders didn't give enough hecks to include all organs

**Comment by ComicIronic** [comment]
You call this atomic?

**Comment by Exxion** [comment]
He actually didn't

**Comment by ComicIronic** [comment]
He insinuated it by making a PR.

**Comment by 9600bauds** [comment]
If bugfixes can be nonatomic, refactors can be nonatomic too

File: code/modules/mob/living/carbon/carbon.dm [hunk 4]
```diff
@@ -31,11 +31,13 @@
 		visible_message("<span class='warning'><B>[M]</B> [M.attacktext] \the [src] !</span>")
 		M.attack_log += text("\[[time_stamp()]\] <font color='red'>attacked [src.name] ([src.ckey])</font>")
 		src.attack_log += text("\[[time_stamp()]\] <font color='orange'>was attacked by [M.name] ([M.ckey])</font>")
+
 		var/damage = rand(M.melee_damage_lower, M.melee_damage_upper)
-		var/dam_zone = pick(LIMB_CHEST, LIMB_LEFT_HAND, LIMB_RIGHT_HAND, LIMB_LEFT_LEG, LIMB_RIGHT_LEG)
+		var/dam_zone = pick(organs_by_name) //Attack a random organ
```
Comment (Intigracy) [inline code/modules/mob/living/carbon/carbon.dm]: won't these `var/whatever = pick(organs_by_name)` return internal organs?

File: code/__DEFINES/reagents.dm [hunk 5]
```diff
@@ -0,0 +1,334 @@
+//Reagents
+
```
Comment (Kurfursten) [inline code/__DEFINES/reagents.dm:2]: Even though this is going to give me a bunch of conflicts, it is so much better in the long run.

File: code/modules/mob/living/carbon/carbon.dm (hunk 4 above)
Comment (unid15) [inline code/modules/mob/living/carbon/carbon.dm]: The organs_by_name list only contains external organs

**Comment by unid15** [comment]
Tested and this doesn't break anything

It's not atomic but then again it doesn't add content or even fix bugs - it just makes the code better

File: code/__DEFINES/reagents.dm [hunk 6]
```diff
@@ -0,0 +1,334 @@
...
+#define VAPORSALT			"vaporsalt"
```
Comment (PJB3005) [inline code/__DEFINES/reagents.dm:3]: > using tabs for alignment.

Comment (unid15) [inline code/__DEFINES/reagents.dm:3]: i copy pasted it
don't start shit

P.S. tabs for life

Comment (PJB3005) [inline code/__DEFINES/reagents.dm:3]: > i copy pasted it
> don't start shit

m8, it's my job to start shit.

> P.S. tabs for life

Wrong. Tabs for indentation, spaces for alignment.

Comment (clusterfack) [inline code/__DEFINES/reagents.dm]: Stop being austistic about the things that don't matter, it's riri tier

Comment (9600bauds) [inline code/__DEFINES/reagents.dm:3]: Yeah, leave some for Comic

Comment (PJB3005) [inline code/__DEFINES/reagents.dm:3]: > Don't matter.

m8

**Comment by clusterfack** [comment]
Dylan has scarred me for life on the phrase 'code cleanup'

**Pull request merged**